src/kernels/kernel.ts

```diff
@@ -250,9 +250,9 @@ abstract class BaseKernel implements IBaseKernel {
                 this._interruptPromise = undefined;
             }
         } finally {
-            Promise.all(
-                Array.from(this.hooks.get('interruptCompleted') || new Set<Hook>()).map((h) => h())
-            ).ignoreErrors();
+            Promise.all(Array.from(this.hooks.get('interruptCompleted') || new Set<Hook>()).map((h) => h())).catch(
```

Comment:
Doesn't this need to be awaited?

Reply:
No, thats why we are swallowing the errors in the promises here, we do not wait to await.

Reply:
Thanks, fixed.